packages/block-library/src/classic/edit.js

```diff
@@ -70,13 +70,13 @@ export default class ClassicEdit extends Component {
 		} = this.props;
 
 		const editor = window.tinymce.get( `editor-${ clientId }` );
-		const currentContent = editor.getContent();
+		const currentContent = editor?.getContent();
 
 		if (
 			prevProps.attributes.content !== content &&
 			currentContent !== content
 		) {
-			editor.setContent( content || '' );
+			editor?.setContent( content || '' ); // eslint-disable-line no-unused-expressions
```

Comment:
This will not be needed as this could occur on load where no changes have been made, so it will never reach this and fail.